eng/Versions.props

```diff
@@ -40,7 +43,7 @@
     <FSharpCoreShippedPackageVersionValue>9.0.300</FSharpCoreShippedPackageVersionValue>
     <!-- -->
     <!-- The pattern for specifying the preview package -->
-    <FSharpCorePreviewPackageVersionValue>$(FSCorePackageVersionValue)-$(PreReleaseVersionLabel).*</FSharpCorePreviewPackageVersionValue>
+    <FSharpCorePreviewPackageVersionValue>$(FSCorePackageVersionValue)-$(PreReleaseVersionLabel).$(FSharpPreReleaseIteration).*</FSharpCorePreviewPackageVersionValue>
```

Comment:
@mmitche :
Is that correct?

`PreReleaseVersionLabel` would be `preview$6`
`FSharpPreReleaseIteration` is `6`


Is `10.0.100-preview$6.6.*` intended, with the two `6` ?